### DISCUSSION TITLE
Just a question...

### DISCUSSION THREAD

**Discussion by arnimattr** [starship/starship#1980] [Q&A] [answered]
# What OS/GTK Theme if Linux is used in the video/gif on the [starship site](https://starship.rs/) ?

 ### Because it looks really nice but not like MacOS Big Sur / Catalina but also not like the GTK Themes I know...

#### Also, you may recommend your favorite gtk themes here :D

**Comment by matchai** [comment, marked as answer]
Hey Arnim,

The configuration used for the gif on the site is documented here:
https://starship.rs/faq/#what-is-the-configuration-used-in-the-demo-gif
